src/test.ts

```diff
@@ -69,4 +69,11 @@ describe("test", () => {
             })
         assert.strictEqual(source, result)
     })
+    it("asMutable", () => {
+        interface X { readonly a: string; readonly b?: number }
+        const x: X = { a: "ee" }
+        const mx = ps.asMutable(x)
+        mx.a = "hello"
+        mx.b = undefined
```

Comment:
It'd probably good to assert that the assignment has worked.

Reply:
https://github.com/ts-common/property-set/blob/master/src/test.ts#L78

Reply:
Ah, I missed that.